src/chainstate/stacks/db/blocks.rs

```diff
@@ -4753,6 +4754,114 @@ impl StacksChainState {
         all_receipts
     }
 
+    /// Process any Delegate-related bitcoin operations
```

Comment:
what does it mean to "process"? briefly describe how the output relates to the input